src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -356,17 +354,20 @@ where
                                 }
                             }
                         }
-                        // TODO: Report these errors to the Healthchecker!
                         Some(Err(e)) => {
```

Comment:
to check my understanding: now we only expect definite errors to be returned from the `source_stream` right?

Reply:
Yes, that's right! In the past only `Fatal` errors made it this far.